### ISSUE TITLE
Error bars are plotted over the data points rather than having the scatter point over the bar [not urgent]

### ISSUE THREAD

**Issue by twVolc** [twVolc/PyCamPermanent#27]
This seems to be an issue that has come from the fact we are now using a newer version of `matplotlib` than the one I had used to develop most of the code (i.e. things plotted more nicely previously). Error bars now seem to plot over the scatter points, both in the spectrometer time series and the DOAS-camera calibration scatter plot. As a result the plots don't look as tidy as for the most part we can't see the blue scatter point itself - just the associated error bar. I wonder if there's a quick way of forcing error bars to the back of the plots or bringing the scatter points forwards?

**Comment by ubdbra001** [comment]
Probably fixable by adjusting the zorder: https://matplotlib.org/stable/gallery/misc/zorder_demo.html

**Comment by ubdbra001** [comment]
This may be a bit tricker, as it looks like the code for building the plot is in `pyplis`.

**Comment by twVolc** [comment]
This is also the case in the DOAS time series figure and may be nice to fix it there too

**Comment by twVolc** [comment]
Yep that's the one. It's just a small thing, but I think with certain plots it might look tidier with them behind - e.g. here (ignore the red LDF line)
![doas_timeseries](https://github.com/twVolc/PyCamPermanent/assets/19552251/14b47c49-3aa6-4e0d-ab78-0641e4e866ba)

**Comment by ubdbra001** [comment]
Okay, I think #91 will fix it, but double check it looks okay to you.
